### PR TITLE
[Storage] `az storage blob list`: Fix next_marker issue 

### PR DESCRIPTION
**Pull request by Juliehzl**
**Description**  

ICM https://portal.microsofticm.com/imp/v3/incidents/details/199846087/home

**Testing Guide**  

`az storage blob list`

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S174

File: src/azure-cli/azure/cli/command_modules/storage/_params.py [hunk 1]
```diff
@@ -146,6 +146,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     timeout_type = CLIArgumentType(
         help='Request timeout in seconds. Applies to each call to the service.', type=int
     )
+    marker_type = CLIArgumentType(
+        help='A string value that identifies the portion of the list of containers to be '
+             'returned with the next listing operation. The operation returns the NextMarker value within '
+             'the response body if the listing operation did not return all containers remaining to be listed '
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/storage/_params.py:152]: Is there example that includes this argument?
Which value can the user can provide?

File: src/azure-cli/azure/cli/command_modules/storage/operations/blob.py [hunk 2]
```diff
@@ -52,6 +54,20 @@ def delete_container(client, container_name, fail_not_exist=False, lease_id=None
...
 
 
+def list_blobs(client, container_name, prefix=None, num_results=None, include=None, delimiter=None, marker=None,
+               timeout=None):
+
+    generator = client.list_blobs(container_name=container_name, prefix=prefix, num_results=num_results,
+                                  include=include, delimiter=delimiter, marker=marker, timeout=timeout)
+    result = list(generator)
+
+    if getattr(generator, 'next_marker', None):
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/storage/operations/blob.py:64]: why will it show warning here?

**Review by jsntcy — APPROVED** [review]
:shipit:

File: src/azure-cli/azure/cli/command_modules/storage/_params.py (hunk 1 above)
Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/storage/_params.py:152]: discussed offline

File: src/azure-cli/azure/cli/command_modules/storage/operations/blob.py (hunk 2 above)
Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/storage/operations/blob.py:64]: discussed offline

**Pull request merged**